src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,101 @@
+.sidebar {
```

Comment:
Nit: Let's drive this layout with a column-based flexbox.  The menu item class can then use `flex-grow: 1` instead of `height: 100%`.  Achieves the same effect, but I think it shows the intent of the style better.